### PR TITLE
Update references to whonix (16->17) and Fedora (37->38)

### PR DESCRIPTION
**Pull request by rocodes**
(Staying in draft mode til changes land)

Refs https://github.com/freedomofpress/securedrop-workstation/issues/934

Also threw in an update to fedora (37-> 38) since that was out of date as well

### CODE REVIEW

File: docs/admin/troubleshooting_updates.rst [hunk 1]
```diff
@@ -107,13 +107,13 @@ Note that ``dom0`` and ``apply_dom0`` are separate steps.
    security-sensitive, and may requrie a reboot to take
    effect.
 
-``sd-*-template`` or ``whoix-gw-16`` update failures
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``sd-*-template`` or ``whoix-gateway-17`` update failures
```
Comment (nathandyer) [inline docs/admin/troubleshooting_updates.rst:110]:
```suggestion
``sd-*-template`` or ``whonix-gateway-17`` update failures
```

Looks like we were/are missing an "n" in whonix here - unless that was intentional for some reason?

**Review by nathandyer — CHANGES_REQUESTED** [review]
This is looking great! One possible typo (lingering from a previous typo), but otherwise LGTM.

File: docs/admin/troubleshooting_updates.rst (hunk 1 above)
Comment (rocodes) [reply docs/admin/troubleshooting_updates.rst]: thank you, fixed in [ec32763](https://github.com/freedomofpress/securedrop-workstation-docs/pull/180/commits/ec32763a40f2ce151c349771c4ecf1086f87d743)

Comment (nathandyer) [reply docs/admin/troubleshooting_updates.rst]: Perfect, thank you!

**Review by nathandyer — APPROVED** [review]
With the latest typo fix, everything LGTM! Approving, but not merging until the Whonix update lands.

**Comment by nathandyer** [comment]
@rocodes Is this ready to be flipped out of draft mode? If so, I'm happy to get this merged today.

**Pull request merged**